conda-recipe/meta.yaml

```diff
@@ -22,19 +22,19 @@ requirements:
   build:
     - python {{PY_VER}}*
     - cmake
-    - boost-cpp 1.66.0  # the boost version needs to be fixed, because we are linking against specific shared objects
-    - xtensor >=0.17.1,<0.18
-    - xtensor-python >=0.20,<0.21
-    - pybind11 >=2.2.1
+    - boost-cpp 1.68.0  # the boost version needs to be fixed, because we are linking against specific shared objects
+    - xtensor 0.18.1
+    - xtensor-python 0.21.0
+    - numpy>=1.15
```

Comment:
FYI generally a good idea to include a space after the package name and before the version comparison. While this *may* work, there is a lot of logic in `conda` and `conda-build` that assumes it can split these components on whitespace.

Reply:
Thanks for the comment, will fix it in the master.